apps/apple/FluentUITester/macos/FluentUITester-macOS/Base.lproj/Main.storyboard

```diff
@@ -690,6 +690,7 @@
                         <windowPositionMask key="initialPositionMask" leftStrut="YES" rightStrut="YES" topStrut="YES" bottomStrut="YES"/>
                         <rect key="contentRect" x="196" y="240" width="480" height="270"/>
                         <rect key="screenRect" x="0.0" y="0.0" width="1680" height="1027"/>
+                        <value key="minSize" type="size" width="480" height="270"/>
```

Comment:
Should we just add this from the AppDelegate instead to keep configuration wherever possible in code for easier maintenance? Or since some window stuff is already here we make all static window stuff here?

Reply:
Vote to keep it here as FRNAppDelegate is quite empty at the moment, and the rest of the static window code here as well.